catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -99,26 +101,203 @@
         <bundle>mvn:ddf.catalog.core/catalog-core-metacardgroomerplugin/${project.version}</bundle>
     </feature>
 
     <feature name="catalog-core-content" version="${project.version}"
              description="Core features for content">
         <bundle>mvn:ddf.catalog.core/catalog-core-contentresourcereader/${project.version}</bundle>
         <bundle>mvn:org.codice.ddf/checksum/${project.version}</bundle>
+    </feature>
+
+    <feature name="catalog-rest" version="${project.version}">
+        <feature>catalog-core-attachment</feature>
+        <feature>catalog-rest-endpoint</feature>
+    </feature>
+
+    <feature name="catalog-rest-endpoint" version="${project.version}"
+             description="REST Endpoint provides CRUD operations on the Catalog Framework.">
+        <bundle>mvn:ddf.catalog.rest/catalog-rest-api/${project.version}</bundle>
+        <bundle>mvn:ddf.catalog.rest/catalog-rest-service-impl/${project.version}</bundle>
+        <bundle>mvn:ddf.catalog.rest/catalog-rest-impl/${project.version}</bundle>
+        <bundle>mvn:ddf.catalog.rest/catalog-rest-endpoint/${project.version}</bundle>
+    </feature>
+
+    <feature name="catalog-opensearch" version="${project.version}">
+        <feature>catalog-opensearch-endpoint</feature>
+        <feature>catalog-opensearch-source</feature>
+    </feature>
+
+    <feature name="catalog-opensearch-endpoint" version="${project.version}"
+             description="OpenSearch Query Endpoint with Atom Transformer.">
+        <bundle>mvn:ddf.catalog.opensearch/catalog-opensearch-endpoint/${project.version}</bundle>
+        <feature>abdera</feature>
+        <bundle>mvn:org.apache.abdera/abdera-extensions-opensearch/${abdera.version}</bundle>
+        <bundle>mvn:org.apache.abdera/abdera-server/${abdera.version}</bundle>
+    </feature>
+
+    <feature name="catalog-opensearch-source" version="${project.version}"
+             description="OpenSearch Federated Source queries a Federated Source using OpenSearch.">
+        <feature>abdera</feature>
+        <feature>cxf-rs-security-sso-saml</feature>
+        <feature>catalog-transformer-geoformatter</feature>
+        <bundle>mvn:org.apache.abdera/abdera-server/${abdera.version}</bundle>
+        <bundle>mvn:ddf.catalog.opensearch/catalog-opensearch-source/${project.version}</bundle>
+    </feature>
+
+    <feature name="abdera" version="${project.version}"
+             description="Apache Abdera">
+        <bundle>wrap:mvn:org.apache.james/apache-mime4j-core/0.7.2</bundle>
+        <bundle>mvn:commons-codec/commons-codec/1.4</bundle>
+        <!-- This bundle <bundle>mvn:org.apache.ws.commons.axiom/axiom-impl/1.2.10</bundle>
+        does not export any classes. Must use the servicemix version. -->
+        <bundle>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.axiom-impl/1.2.12-2</bundle>
+        <bundle>mvn:org.apache.ws.commons.axiom/axiom-api/1.2.14</bundle>
```

Comment:
✏️ Should be able to use the property `axiom.version` (defined in `/catalog.pom.xml`).